CONTRIBUTING.md

```diff
@@ -1,50 +1,60 @@
 # Contributing to osquery
 
 We want to make contributing to osquery as simple and transparent as possible. These guidelines
-explain the basics of the osquery development process and how you can contribute too. Please read
+explain the basics of the osquery development process and how you can contribute. Please read
 these guidelines before submitting your code as they are designed to save you time later on when
-your code is under review and give you the basics of how to get started.
+your code is under review.
 
 ## Contributing 101
 
-All contributions from the community are submitted via pull requests
+All contributions from the community are submitted via pull requests (PRs)
 open against the osquery's
 [master](https://github.com/osquery/osquery/tree/master) branch on
-GitHub. After being reviewed by the core team and tested by CI. If all
+GitHub. After being reviewed by the _core team_ and tested by CI, if all
 is well, they will be pushed to master and the corresponding PR closed.
 
+You can see who the _core team_ is by viewing the [team page](https://github.com/orgs/osquery/teams)
+on the osquery GitHub organization.
+
 If you need help, both the core team and community members are on the osquery
 [Slack](https://osquery.slack.com). Feel free to register using the following
 [link](https://slack.osquery.io/) if you haven't done so yet and get in touch with us.
+The `#code-review` Slack channel has been set up to handle urgent review needs as well as questions
+about your PR. Note: prefer to keep (or echo) discussion about code changes in the GitHub
```

Comment:
I suspect "echo" isn't meaningful to most people. I think the paragraph is great without the parenthetical.